templates/flasher-configmap.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: flasher-config
+  namespace: default
+data:
+  config.yaml: |
+    log_level: debug
+    app_kind: worker
+    concurrency: 5
+    store_kind: serverservice
+    serverservice:
+      endpoint: http://serverservice:8000
+      disable_oauth: true
+      facility_code: dc13
+    events_broker_kind: nats
+    nats:
+      url: nats://nats:4222
+      app_name: flasher
+      publisher_subject_prefix: com.hollow.sh.controllers.responses
+      stream_urn_ns: hollow-controllers
+      #subscribe_subjects:
+      #  - com.hollow.sh.controllers.commands.>
+      connect_timeout: 30s
+      #creds_file:
+      consumer:
+        pull: true
```

Comment:
The orchestrator and the controllers in the sandbox are configured as consumers as a `queue group` using [workQueue](https://natsbyexample.com/examples/jetstream/workqueue-stream/go), and based on various texts, this setup should ideally be deployed in `pull` mode instead of `push` for a few reasons, 
  - horizontal scalability
  - messages ordering
  - avoids duplicates
  - avoids thundering herd issues when a client has just come online. 

From https://natsbyexample.com/examples/jetstream/push-consumer/go/
```
In the happy path, the subscription receiving these messages can keep up and acknowledge the message prior to the ack wait time has elapsed.

Where push consumers can get unwieldy and confusing is when the subscriber cannot keep up, message processing errors start occurring, or the active subscription gets interrupted. Messages start getting redelivered and being interleaving with new messages pushed from the stream.

In practice, ephemeral push consumers can be a lightweight and useful way to do one-off consumption of a subset of messages in a stream. However, if you have a durable use case, it is recommended to access [pull consumers](https://natsbyexample.com/examples/jetstream/pull-consumer/go/) first which provides more control and implicit support for scaling out consumption.
```